events/nats_config.go

```diff
@@ -66,6 +66,14 @@ type NatsOptions struct {
 
 	// Setting Stream parameters will cause a NATS stream to be added.
 	Stream *NatsStreamOptions `mapstructure:"stream"`
+
+	// Nats KV parameters
+	KV *NatsKVOptions `mapstructure:"kv"`
```

Comment:
Why add this here?

Reply:
mainly for consistency in loading the NATS config from the env/config file 
https://github.com/metal-toolbox/sandbox/pull/26/files#diff-4d55b7b2bd04c8f271ca9fb638669a82d8d00aba55ccc6bde46aafd0c72e9236R32
 
Also I'd like to remove the `--replica-count` cli flag since its an internal thing and I don't want all controllers to have to specify this through the cli -  https://github.com/metal-toolbox/flasher/blob/aa6c0eb71ddafea5878008d0ad8c193c75580464/cmd/run.go#L119

Reply:
The only reason we'd change replica count is to support running in the sandbox or in other integration tests. I don't think this should be configurable.